src/lib/common/limits.h

```diff
@@ -75,4 +73,89 @@
 #define ALARM_MGR_NOTIFICATION_ERROR_LOG_SAMPLING     1
 #define ALARM_MGR_BAD_INPUT_LOG_SAMPLING              1
 
+
+/* ****************************************************************************
+*
+* Message definitions - 
+*/
+#define MAX_STA_MSG_SIZE (20 * 1024)           // 20 KB (HTTP request static buffer)
+#define MAX_DYN_MSG_SIZE (8 * 1024 * 1024)     // 8 MB  (maximum length of the HTTP request dynamic buffer)
+
+
+/* ****************************************************************************
+*
+* PAYLOAD_MAX_SIZE - 
+*/
+#define PAYLOAD_MAX_SIZE   (1 * 1024 * 1024) // 1 MB Maximum size of the payload
+
+
+/* ****************************************************************************
+*
+* IP - 
+*/
+#define  LOCAL_IP_V6  "::"
+#define  LOCAL_IP_V4  "0.0.0.0"
+
+
+/* ****************************************************************************
+*
+* STATIC_BUFFER_SIZE - to avoid mallocs for "smaller" requests
+*/
+#define STATIC_BUFFER_SIZE (32 * 1024) // 32 KB 
+
+
+/* ****************************************************************************
+*
+* CONSTANTS RESTINIT - 
+*/ 
+#define   CONNECTION_MEM  	64   
```

Comment:
I'd suggest to use 

DEFAULT_CONNECTION_MEM
DEFAULT_MAX_CONNECTIONS
DEFAULT_MHD_THREAD_POOL

@kzangeli , what do you think?